packages/configurations/typescript.eslint-config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
       rules: {
         '@typescript-eslint/unbound-method': 0,
         '@typescript-eslint/no-namespace': 0,
-        'import/extensions': [2, 'always'],
+        'import/extensions': [2, 'ignorePackages'],
```

Comment:
let's remove this rule from both overrides and add it to `eslint-config.js` instead.
This should not break anything as the new rule config is more permissive.